test/expect/TestBatched.test_if_else_with_scalar.expect

```diff
@@ -8,7 +8,7 @@ graph(%a.1_data : Dynamic
   %7 : Float() = prim::NumToTensor(%6)
   %other : float = prim::TensorToNum(%7)
   %9 : Dynamic = aten::gt(%a.1_data, %other)
-  %10 : int = prim::TensorToNum(%9)
+  %10 : bool = prim::TensorToBool(%9)
```

Comment:
`%10` here seems to be dead (it's not used in this graph anywhere), is this a bug or does DCE not run on batched graphs?